test/expected/create_hypertable.out

```diff
@@ -235,14 +235,169 @@ ERROR:  column "location" is already a dimension
+----------+---------------+--------------+--------------------+-----------------------+------------------+------------------+------------------
+        2 |             5 |           12 |                  3 | _timescaledb_internal | _hyper_5_2_chunk | 1097366400000000 | 1097452800000000
+        3 |             5 |           12 |                  4 | _timescaledb_internal | _hyper_5_3_chunk | 1098230400000000 | 1098316800000000
+(2 rows)
+
+SELECT * FROM test.show_constraints('_timescaledb_internal._hyper_5_2_chunk');
+  Constraint  | Type | Columns | Index |                                                                      Expr                                                                      | Deferrable | Deferred | Validated 
+--------------+------+---------+-------+------------------------------------------------------------------------------------------------------------------------------------------------+------------+----------+-----------
+ constraint_3 | c    | {time}  | -     | (("time" >= 'Sat Oct 09 17:00:00 2004 PDT'::timestamp with time zone) AND ("time" < 'Sun Oct 10 17:00:00 2004 PDT'::timestamp with time zone)) | f          | f        | t
+(1 row)
+
+SELECT * FROM test.show_constraints('_timescaledb_internal._hyper_5_3_chunk');
+  Constraint  | Type | Columns | Index |                                                                      Expr                                                                      | Deferrable | Deferred | Validated 
+--------------+------+---------+-------+------------------------------------------------------------------------------------------------------------------------------------------------+------------+----------+-----------
+ constraint_4 | c    | {time}  | -     | (("time" >= 'Tue Oct 19 17:00:00 2004 PDT'::timestamp with time zone) AND ("time" < 'Wed Oct 20 17:00:00 2004 PDT'::timestamp with time zone)) | f          | f        | t
+(1 row)
+
+-- add dimension to the existing chunks by adding -inf/inf dimension slices
+SELECT add_dimension('dim_test', 'device', 2);
+         add_dimension         
+-------------------------------
+ (13,public,dim_test,device,t)
+(1 row)
+
+SELECT * FROM test.show_constraints('_timescaledb_internal._hyper_5_2_chunk');
+  Constraint  | Type | Columns | Index |                                                                      Expr                                                                      | Deferrable | Deferred | Validated 
+--------------+------+---------+-------+------------------------------------------------------------------------------------------------------------------------------------------------+------------+----------+-----------
+ constraint_3 | c    | {time}  | -     | (("time" >= 'Sat Oct 09 17:00:00 2004 PDT'::timestamp with time zone) AND ("time" < 'Sun Oct 10 17:00:00 2004 PDT'::timestamp with time zone)) | f          | f        | t
```

Comment:
There is no CHECK constraint for the `device` dimension. I would expect one to exist for each chunk after adding a new dimension, at least for new chunks. Should probably create a new chunk after add_dimension() to show that this is the case.

Reply:
This part of test does not show them for previous chunks, we did not add full range CHECK constraints before. This parts shows change in the `dimension_slice` table for old chunks.

What you are asking for is down below basically, when we add a new chunk after that (starting from line 306):

Well basically it shows them down below starting from here: https://github.com/timescale/timescaledb/pull/4495/files/63aed2332487bb4a07911438f169c72fb67eec4b#diff-7a9fb9a68eb195be632aa06cc899ef25fa305fb55643dfc9c2d0727624aa3a25R306